lib/Julianable.rb

```diff
@@ -0,0 +1,37 @@
+module Julianable
+
+  def lowest_scoring_visitor
+    # visiting team with the lowest average score across all seasons
+    # take away_team_id and calculate average of scores over pre and reg seasons
+    # sort for lowest average score
+  end
+  
+  def highest_scoring_visitor
+    # visiting team with the highest average score across all seasons
+    # take away_team_id and calculate average of scores over pre and reg seasons
+    # sort for highest average score
+  end
+
+  def count_of_teams
+    # count the teams in the teams class
+    # @team_id.count
+  end
+
+  def percentage_home_wins
+    # percetage of games that a home_team has won
+    # find total games played in games class. take team_id.HoA(home).win
+    # team_id.HoA(home).win/ total_games played .round(2)
+  end
+  
+  def percentage_visitor_wins
+    # percetage of games that a away_team has won
+    # find total games played in games class. take team_id.HoA(away).win
+    # team_id.HoA(away).win/ total_games played .round(2)
+  end
+  
+  def percentage_ties
+    # percetage of games that end in a tie
+    # find total games played in games class. find tie_games
+    # tie_games/ total_games .round(2)
+  end
+end
```

Comment:
Do we want to add this file if all of our assigned methods are going into StatTracker?

Reply:
Yes, thanks for resolving this.  Late night pushes diminish critical thinking.